test/node/utils/sets_to_arrays.js

```diff
@@ -0,0 +1,13 @@
+
+// Proclaim appears to be unable to compare sets in a deepEqual
```

Comment:
@triblondon Could you open an issue on the proclaim repo for this? We can definitely make time to update proclaim to work with ES6 objects.